recipe/meta.yaml

```diff
@@ -10,19 +10,19 @@ source:
   sha256: ae94047c79b4f48462b66981f66a67b6a833d75d4c40e7afead491b1865f1142
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win]
 
 requirements:
   build:
+    - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - cmake
     - make
     - pkg-config
 
   host:
-    - grpc-cpp
-    - fmt
+    - grpc-cpp  {{ grpc_cpp }}
```

Comment:
You don't need the jinja2 here I think.

Reply:
grpc-cpp is not a valid key. because of the `-`